vmdb/app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -292,12 +292,12 @@ def build_pxe_img_grid(pxe_imgs, sort_order = nil, sort_by = nil)
   end
 
   def build_iso_img_grid(iso_imgs,sort_order="ASC",sort_by="name")
-    @edit[:iso_img_sortdir] = sort_order
     @edit[:iso_img_headers] = {
         "name"=>"Name"
-    }                                                         #Using it to get column display name on screen to show sort by
-    @edit[:iso_img_columns] = ["name"] #Using it to get column names for sort
-    @edit[:iso_img_sortcol] = sort_by                 # in case sort column is not set, set the defaults
+    }                                                       # Using it to get column display name on screen to show sort by
+    @edit[:iso_img_columns] = ["name"]                      # Using it to get column names for sort
+    @edit[:iso_img_sortcol] = sort_by    ||= "name"         # in case sort column is not set, set the defaults
+    @edit[:iso_img_sortdir] = sort_order ||= "ASC"
```

Comment:
If we need the `||=`, can we put it on a different line?  Or can we fix the callers not to pass `nil`?  Adding the `||=` seems like fixing a symptom rather than the problem.